app.js

```diff
@@ -25,20 +32,77 @@ app.get('/', (req, res) => {
 //  POST  /recipes route
 
 
+app.post('/recipes', (req,res) => {
+
+    Recipe.create(req.body)
+    .then((createdRecipe)=>{
+        res.status(201).json(createdRecipe)
+    })
+    .catch((err)=>{
+        res.status(500).json({message:"Error, recipe not included"})
+    })
+
+
+
+})
+
 //  Iteration 4 - Get All Recipes
 //  GET  /recipes route
 
+app.get('/recipes', (req,res) => {
+
+   Recipe.find()
+   .then((allRecipes)=>{
+    res.status(200).json(allRecipes)
+   })
+   .catch((err)=>{
+    res.status(500).json({message:"Error, recipes won't be shown"})
+   })
+   
+
+
+})
 
 //  Iteration 5 - Get a Single Recipe
 //  GET  /recipes/:id route
 
+app.get('/recipes/:id' , (req,res)=>{
+    Recipe.findById(req.params.id)
```

Comment:
also applys for the id in the params 
```suggestion
const { id } = req.params
Recipe.findById(id)
```